generators/entity-client/templates/common/src/test/javascript/cypress/integration/entity/entity.spec.ts.ejs

```diff
@@ -166,10 +166,7 @@ describe('<%= entityClass %> e2e test', () => {
 
     <%_ } else if (fieldType === 'Boolean') { _%>
     cy.get(`[data-cy="<%= fieldName %>"]`).should('not.be.checked');
-    if ('<%= fieldValue %>' === 'true') {
-        cy.get(`[data-cy="<%= fieldName %>"]`).click().should('be.checked');
-    }
-
+    cy.get(`[data-cy="<%= fieldName %>"]`).click().should('be.checked');
```

Comment:
This will loose randomization
```suggestion
    <%_ if (fieldValue) {  _%>
    cy.get(`[data-cy="<%= fieldName %>"]`).click().should('be.checked');
    <%_ } _%>
```

Reply:
It is necessary because some entites need to have their boolean to be true

Reply:
Why?

Reply:
When we try to create entity's instance, sometimes tests failed because it can't validate creation because of a boolean field which is not checked

Reply:
So, we think that is better to always check boolean's field to avoid problem with required booleans

Reply:
It’s a bug at the client side so.
The model should be created with false value if the field is required.
Or we should switch to a radio instead of a checkbox.
Anyway I think you can open an issue to discuss it, WYT?
Updated the suggestion

Reply:
IMO requiring the user to click the checkbox, falsely implies the checkbox must be true.

Reply:
This is cypress discovering bugs 😄

Reply:
There is so many cases right here with entities :sweat_smile:

Reply:
> It’s a bug at the client side so.
> The model should be created with false value if the field is required.
> Or we should switch to a radio instead of a checkbox.
> Anyway I think you can open an issue to discuss it, WYT?
> Updated the suggestion

For the moment, I can't reproduce the error. So, let's not open an issue for that

Reply:
My bad, I think it is probably link to this issue [jhipster#12296](https://github.com/jhipster/generator-jhipster/pull/12296).

Reply:
@nassimerrahoui can we use a randomized 1 click then?

Reply:
IMO, as we are not sure that the problem will not occur next times, I think it is more safe to always keep the click